modules/nw-dpu-running-workloads.adoc

```diff
@@ -48,7 +48,13 @@ spec:
         openshift.io/dpu: '1'
 ----
 +
-<1> The name of the node where the workload is deployed.
+* `spec.nodeSelector`: The node selector schedules the pod on the node with the DPU resource. You can use any standard Kubernetes selector for this, such as `kubernetes.io/hostname` to target a specific node as shown in the example YAML.
```

Comment:
Add a comma after  `kubernetes.io/hostname`:
You can use any standard Kubernetes selector for this, such as `kubernetes.io/hostname`**,** to target a specific node as shown in the example YAML.